README.md

```diff
@@ -2,7 +2,18 @@
 
 ## DESCRIPTION
 
-## Environment Setup
+#### Environment Setup
+This project is training and testing on Ubuntu 18.04 with Anaconda. We use one Quatro RTX 6000 and two RTX 2070 Super to train our model. Because all the video cards we use are RTX series, we can enable the RTX 16-bit optimization which makes our training faster. The model is built on Keras. The training is on blocks and patches. We use three framesets at a time, which is 9 frames in total. The block memory we use is about 22 GB and patch memory is about 6 GB.
```

Comment:
We train ... 
... The model is built on Keras... 
The training is on blocks and patches. 
frame sets
which have 
why 22GB and 6GB: 1 pixel -> block & patch

Reply:
mini-batch